### PR TITLE
bugfix/46-fix-edit-icon-on-linkbox-component

### PR DESCRIPTION
**Pull request by bmirra01**
Sorted by adding the code "event.preventDefault()" to avoid having the link opening event triggered everytime the button edit and delete is pressed. Also added this fix on Add button in case this is needed in the future for other components.

#46

### CODE REVIEW

File: packages/lumina-core/src/components/action-buttons/add/index.tsx [hunk 1]
```diff
@@ -26,7 +26,8 @@ export const AddComponentButton = ({
   const { handleToggleMenu } = useToggleMenuContext()
   const { handleOpenModal } = useToggleModalContext()
 
-  const handleButtonClick = () => {
+  const handleButtonClick = (event: any) => {
```
Comment (JWebCoder) [inline packages/lumina-core/src/components/action-buttons/add/index.tsx:29]: Do not use any please.

File: packages/lumina-core/src/components/action-buttons/delete/index.tsx [hunk 2]
```diff
@@ -23,7 +23,8 @@ export const DeleteComponentButton = ({
   const { handleOpenModal } = useToggleModalContext()
   const { handleToggleMenu } = useToggleMenuContext()
 
-  const handleToggleDeleteModal = () => {
+  const handleToggleDeleteModal = (event: any) => {
```
Comment (JWebCoder) [inline packages/lumina-core/src/components/action-buttons/delete/index.tsx:26]: Do not use any please.

File: packages/lumina-core/src/components/action-buttons/edit/index.tsx [hunk 3]
```diff
@@ -32,7 +32,8 @@ export const EditComponentButton = ({
   const { handleOpenModal } = useToggleModalContext()
   const { handleToggleMenu } = useToggleMenuContext()
 
-  const handleToggleEditModal = () => {
+  const handleToggleEditModal = (event: any) => {
```
Comment (JWebCoder) [inline packages/lumina-core/src/components/action-buttons/edit/index.tsx:35]: Do not use any please.

**Review by JWebCoder — CHANGES_REQUESTED** [review]

File: packages/lumina-core/src/components/action-buttons/add/index.tsx (hunk 1 above)
Comment (bmirra01) [reply packages/lumina-core/src/components/action-buttons/add/index.tsx]: Replaced it by React.MouseEvent<HTMLButtonElement>

File: packages/lumina-core/src/components/action-buttons/delete/index.tsx (hunk 2 above)
Comment (bmirra01) [reply packages/lumina-core/src/components/action-buttons/delete/index.tsx]: Replaced it by React.MouseEvent<HTMLButtonElement>

File: packages/lumina-core/src/components/action-buttons/edit/index.tsx (hunk 3 above)
Comment (bmirra01) [reply packages/lumina-core/src/components/action-buttons/edit/index.tsx]: Replaced it by React.MouseEvent<HTMLButtonElement>

**Review by Felix-by-merkle — APPROVED** [review]

**Review by antonio-rodrigues — APPROVED** [review]

**Pull request merged**